themes/bootstrap3/templates/ajax/status-full.phtml

```diff
@@ -33,7 +33,13 @@
         <?php if ($item['use_unknown_message'] ?? false): ?>
           <span class="text-muted"><?=$this->transEsc('status_unknown_message')?></span>
         <?php elseif ((int)$item['availability'] === \VuFind\ILS\Logic\ItemStatus::STATUS_UNAVAILABLE): ?>
-          <span class="text-danger"><?=$this->transEsc($item['status'])?></span>
+	  <span class="text-danger">
+	      <?php if ($item['in_transit'] ?? null): ?>
+                  <?=$this->transEsc('transit_to', ['%%location%%' => $item['in_transit'], '%%date%%' => $item['in_transit_date']])?>
```

Comment:
If you make my suggested change above, this will of course need to be adjusted to match.